PySDM/formulae.py

```diff
@@ -59,92 +77,18 @@ def __init__(  # pylint: disable=too-many-locals
         )
 
-        self.condensation_coordinate = _magick(
-            condensation_coordinate,
-            physics.condensation_coordinate,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.saturation_vapour_pressure = _magick(
-            saturation_vapour_pressure,
-            physics.saturation_vapour_pressure,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.latent_heat = _magick(
-            latent_heat, physics.latent_heat, fastmath, constants, dimensional_analysis
-        )
-        self.hygroscopicity = _magick(
-            hygroscopicity,
-            physics.hygroscopicity,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.drop_growth = _magick(
-            drop_growth, physics.drop_growth, fastmath, constants, dimensional_analysis
-        )
-        self.surface_tension = _magick(
-            surface_tension,
-            physics.surface_tension,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.diffusion_kinetics = _magick(
-            diffusion_kinetics,
-            physics.diffusion_kinetics,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.diffusion_thermics = _magick(
-            diffusion_thermics,
-            physics.diffusion_thermics,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.ventilation = _magick(
-            ventilation, physics.ventilation, fastmath, constants, dimensional_analysis
-        )
-        self.state_variable_triplet = _magick(
-            state_variable_triplet,
-            physics.state_variable_triplet,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.particle_advection = _magick(
-            particle_advection,
-            physics.particle_advection,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.hydrostatics = _magick(
-            hydrostatics,
-            physics.hydrostatics,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.freezing_temperature_spectrum = _magick(
-            freezing_temperature_spectrum,
-            physics.freezing_temperature_spectrum,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
-        self.heterogeneous_ice_nucleation_rate = _magick(
-            heterogeneous_ice_nucleation_rate,
-            physics.heterogeneous_ice_nucleation_rate,
-            fastmath,
-            constants,
-            dimensional_analysis,
-        )
+        for phys in nones:
+            setattr(
+                self,
+                phys,
```

Comment:
What does phys and nones mean?

Reply:
changing to `for component in components`, good point!
each "component" here corresponds to a subdirectory in `PySDM/physics` (adding it as a comment as well)